js/database.js

```diff
@@ -0,0 +1,66 @@
+import { getToday, incrementDay, isSameDate, MILISECOND_HOUR } from "./dateManipulation.js"
+
+if(!localStorage.getItem("eventId")){
+    localStorage.setItem("eventId", 0)
+}
+
+const findByDate = (date) => {
+    const nextEventId = localStorage.getItem("eventId")
+
+    const result = []
+
+    for (let i = 0; i < nextEventId; i++) {
```

Comment:
If you would store form data as suggested in another comment under localStorage key "forms", you could loop like this:

```
forms.forEach(form => {
   ...
})
```

Or in order to filter events which are satifying some condition, you could do it in less code like this:

`const result = forms.filter(form => isSuitableForm(form))`